### DISCUSSION TITLE
《1984》乔治·奥威尔

### DISCUSSION THREAD

**Discussion by GitHeP** [WeNeedHome/SummaryOfLoanSuspension#859] [General]
Big brother is watching at you

**Comment by go2006010215** [comment]
不需要at，Big brother is watching you.

**Reply by purpose1113** [reply]
现实世界没有乌托邦

**Reply by Sora-super** [reply]
> 

语法可以啊

**Comment by hktxt** [comment]
第一部 第一章
四月间，天气寒冷晴朗，钟敲了十三下。温斯顿史密斯为了要躲寒风，紧缩着脖子，很快地溜进了胜利大厦的玻璃门，不过动作不够迅速，没有能够防止一阵沙土跟着他刮进了门。

　　门厅里有一股熬白菜和旧地席的气味。门厅的一头，有一张彩色的招贴画钉在墙上，在室内悬挂略为嫌大了一些。

　　画的是一张很大的面孔，有一米多宽：这是一个大约四十五岁的男人的脸，留着浓密的黑胡子，面部线条粗犷英俊。温斯顿朝楼梯走去。用不着试电梯。即使最顺利的时候，电梯也是很少开的，现在又是白天停电。这是为了筹备举行仇恨周而实行节约。温斯顿的住所在七层楼上，他三十九岁，右脚脖子上患静脉曲张，因此爬得很慢，一路上休息了好几次。每上一层楼，正对着电梯门的墙上就有那幅画着很大脸庞的招贴画凝视着。这是属于这样的一类画，你不论走到哪里，画面中的眼光总是跟着你。下面的文字说明是：老大哥在看着你。

　　在他住所里面，有个圆润的嗓子在念一系列与生铁产量有关的数字。声音来自一块象毛玻璃一样的椭圆形金属板，这构成右边墙壁的一部分墙面。温斯顿按了一个开关，声音就轻了一些，不过说的话仍听得清楚。这个装置(叫做电幕)可以放低声音，可是没有办法完全关上。他走到窗边。

　　他的身材瘦小纤弱，蓝色的工作服——那是党内的制服——

　　更加突出了他身子的单薄。他的头发很淡，脸色天生红润，他的皮肤由于用粗肥皂和钝刀片，再加上刚刚过去的寒冬，显得有点粗糙。

　　外面，即使通过关上的玻璃窗，看上去也是寒冷的。在下面街心里，阵阵的小卷风把尘土和碎纸吹卷起来，虽然阳光灿烂，天空蔚蓝，可是除了到处贴着的招贴画以外，似乎什么东西都没有颜色。那张留着黑胡子的脸从每一个关键地方向下凝视。在对面那所房子的正面就有一幅，文字说朋是：老大哥在看着你。那双黑色的眼睛目不转睛地看着温斯顿的眼睛。在下面街上有另外一张招贴画，一角给撕破了，在风中不时地吹拍着，一会儿盖上，一会儿又露出唯一的一个词儿“英社”。在远处，一架直升飞机在屋预上面掠过，象一只蓝色的瓶子似的徘徊了一会，又绕个弯儿飞走。这是警察巡逻队，在伺察人们的窗户。不过巡逻队并不可怕，只有思想警察才可怕。

　　在温斯顿的身后，电幕上的声音仍在喋喋不休地报告生铁产量和第九个三年计划的超额完成情况。电幕能够同时接收和放送。温斯顿发出的任何声音，只要比极低声的细语大一点，它就可以接收到；此外，只要他留在那块金属板的视野之内，除了能听到他的声音之外，也能看到他的行动。当然，没有办法知道，在某一特定的时间里，你的一言一行是否都有人在监视着。思想警察究竟多么经常，或者根据什么安排在接收某个人的线路，那你就只能猜测了。甚至可以想象，他们对每个人都是从头到尾一直在监视着的。反正不论什么时候，只要他们高兴，他们都可以接上你的线路。你只能在这样的假定下生活——从已经成为本能的习惯出发，你早已这样生活了：你发出的每一个声音，都是有人听到的，你作的每一个动作，除非在黑暗中，都是有人仔细观察的。

　　温斯顿继续背对着电幕。这样比较安全些；不过他也很明白，甚至背部有时也能暴露问题的。一公里以外，他工作的单位真理部高耸在阴沉的市景之上，建筑高大，一片白色。这，他带着有些模糊的厌恶情绪想——这就是伦敦，一号空降场的主要城市，一号空降场是大洋国人口位居第三的省份。他竭力想挤出一些童年时代的记忆来，能够告诉他伦敦是不是一直都是这样的。是不是一直有这些景象：破败的十九世纪房子，墙头用木材撑着，窗户钉上了硬纸板，屋顶上盖着波纹铁皮，倒塌的花园围墙东倒西歪；还有那尘土飞扬、破砖残瓦上野草丛生的空袭地点；还有那炸弹清出了一大块空地，上面忽然出现了许多象鸡笼似的肮脏木房子的地方。可是没有用，他记不起来了；除了一系列没有背景、模糊难辨的、灯光灿烂的画面以外，他的童年已不留下什么记忆了。

　　真理部——用新话来说叫真部——同视野里的任何其他东西都有令人吃惊的不同。这是一个庞大的金字塔式的建筑，白色的水泥晶晶发亮，一层接着一层上升，一直升到高空三百米。从温斯顿站着的地方，正好可以看到党的三句口号，这是用很漂亮的字体写在白色的墙面上的：

　　战争即和平

　　自由即奴役

　　无知即力量。

　　据说，真理部在地面上有三千间屋子，和地面下的结构相等。在伦敦别的地方，还有三所其他的建筑，外表和大小与此相同。它们使周围的建筑仿佛小巫见了大巫，因此你从胜利大厦的屋顶上可以同时看到这四所建筑。它们是整个政府机构四部的所在地：真理部负责新闻、娱乐、教育、艺术；和平部负责战争；友爱部维持法律和秩序；富裕部负责经济事务。用新话来说，它们分别称为真部、和部、爱部、富部。

　　真正教人害怕的部是友爱部．它连一扇窗户也没有。温斯顿从来没有到友爱部去过，也从来没有走近距它半公里之内的地带．这个地方，除非因公，是无法进入的，而且进去也要通过重重铁丝网、铁门、隐蔽的机枪阵地．甚至在环绕它的屏障之外的大街上，也有穿着黑色制服、携带连枷棍的凶神恶煞般的警卫在巡逻。

　　温斯顿突然转过身来．这时他已经使自已的脸部现出一种安详乐观的表情，在面对电幕的时候，最好是用这种表情。他走过房间，到了小厨房里。在一天的这个时间里离开真理部，他牺牲了在食堂的中饭，他知道厨房里没有别的吃的，只有一块深色的面包，那是得省下来当明天的早饭的。

　　他从架子上拿下一瓶无色的液体，上面贴着一张简单白色的标签：胜利杜松子酒。它有一种令人难受的油味儿，象中国的黄酒一样。温斯顿倒了快一茶匙，硬着头皮，象吃药似的咕噜一口喝了下去。

　　他的脸马上绯红起来，眼角里流出了泪水。这玩艺儿象硝酸，而且，喝下去的时候，你有一种感觉，好象后脑勺上挨了一下橡皮棍似的。不过接着他肚子里火烧的感觉减退了，世界看起来开始比较轻松愉快了。他从一匣挤瘪了的胜利牌香烟盒中拿出一支烟来，不小心地竖举着，烟丝马上掉到了地上。他拿出了第二支，这次比较成功。他回到了起居室，坐在电幕左边的一张小桌子前。他从桌子抽屉里拿出一支笔杆、一瓶墨水、一本厚厚的四开本空白簿子，红色的书脊，大理石花纹的封面。

　　不知什么缘故，起居室里的电幕安的位置与众不同。按正常的办法，它应该安在端墙上，可以看到整个房间，可是如今却安在侧墙上，正对着窗户。在电幕的一边，有一个浅浅的壁龛，温斯顿现在就坐在这里，在修建这所房子的时候，这个壁龛大概是打算放书架的。温斯顿坐在壁龛里，尽量躲得远远的，可以处在电幕的控制范围之外，不过这仅仅就视野而言。当然，他的声音还是可以听到的，但只要他留在目前的地位中，电幕就看不到他。一半是由于这间屋子的与众不同的布局，使他想到要做他目前要做的事。

　　但这件事也是他刚刚从抽屉中拿出来的那个本子使他想到要做的。这是一本特别精美的本子。光滑洁白的纸张因年代久远而有些发黄，这种纸张至少过去四十年来已久未生产了。不过他可以猜想，这部本子的年代还要久远得多。他是在本市里一个破破烂烂的居民区的一家发霉的小旧货铺中看到它躺在橱窗中的，到底是哪个区，他已经记不得了。他当时一眼就看中，一心要想得到它。照理党员是不许到普通店铺里去的(去了就是“在自由市场上做买卖”)，不过这条规矩并不严格执行，因为有许多东西，例如鞋带、刀片，用任何别的办法是无法弄到的，他回头很快地看了一眼街道两头，就溜进了小铺子，花二元五角钱把本子买了下来。当时他并没有想到买来干什么用。他把它放在皮包里，不安地回了家。即使里面没有写什么东西，有这样一个本子也是容易引起怀疑的。

　　他要做的事情是开始写日记。写日记并不是不合法的(没有什么事情是不合法的，因为早已不再有什么法律了)，但是如被发现，可以相当有把握地肯定，会受到死刑的惩处，或者至少在强迫劳动营里干苦役二十五年。温斯顿把笔尖愿在笔杆上，用嘴舔了一下，把上面的油去掉。这种沾水笔已成了老古董，甚至签名时也不用了，他偷偷地花了不少力气才买到一支，只是因为他觉得这个精美乳白的本子只配用真正的笔尖书写，不能用墨水铅笔涂划。实际上他已不习惯手书了。除了极简短的字条以外，一般都用听写器口授一切，他目前要做的事，当然是不能用听写器的。他把笔尖沾了墨水，又停了一下，不过只有一刹那。他的肠子里感到一阵战颤。在纸上写标题是个决定性的行动。他用纤小笨拙的字体写道：

　　1984年4月4日

　　他身子往后一靠。一阵束手无策的感觉袭击了他。首先是，他一点也没有把握，今年是不是1984年。大致是这个日期，因为他相当有把握地知道，自已的年龄是三十九岁，而且他相信他是在1944年或1945年生的。但是，要把任何日期确定下来，误差不出一两年，在当今的时世里，是永远办不到的。

　　他突然想到，他是在为谁写日记呀？为将来，为后代。

　　他的思想在本子上的那个可疑日期上犹豫了一会儿，突然想起了新话中的一个词儿“双重思想”。他头一次领梧到了他要做的事情的艰巨性。你怎么能够同未来联系呢？从其性质来说，这样做就是不可能的。只有两种情况，要是未来同现在一样，在这样的情况下未来就不会听他的，要是未来同现在不一样，他的处境也就没有任何意义了。

　　他呆呆地坐在那里，看着本子。电幕上现在播放刺耳的军乐了。奇怪的是，他似乎不仅丧失了表达自己的能力，而且甚至忘掉了他原来要想说什么话了。过去几个星期以来，他一直在准备应付这一时刻，他从来没有想到过，除了勇气以外还需要什么。实际写作会是很容易的。他要做的只是把多年来头脑里一直在想的、无休止的、无穷尽的独白付诸笔墨就行了。但是在目前，甚至独白也枯竭了。此外，他的静脉曲张也开始痒了起来，使人难熬。他不敢抓它，因为一抓就要发炎。时间滴嗒地过去。他只感到面空白的纸张，脚脖子上的皮肤发痒，音乐的聒噪，杜松子酒引起的一阵醉意。

　　突然他开始慌里慌张地写了起来，只是模模糊糊地意识到他写的是些什么。他的纤小而有些孩子气的笔迹在本子上弯弯曲曲地描划着，写着写着，先是省略了大写字母，最后连句号也省略了：

　　1984年4月4日。昨晚去看电影。全是战争片。一部很好，是关于一艘装满难民的船，在地中海某处遭到空袭。观众看到一个大胖子要想游开去逃脱追他的直升飞机的镜头感到很好玩。你起初看到他象一头海豚一样在水里浮沉，后来通过直升飞机的瞄准器看到他，最后他全身是枪眼，四周的海水都染红了，他突然下沉，好象枪眼里吸进了海水一样。下沉的时候观众笑着叫好。接着你看到一艘装满儿童的救生艇，上空有一架直升飞机在盘旋。

　　有个中年妇女坐在船首，大概是个犹太女人，怀中抱着一个大约三岁的小男孩。小男孩吓得哇哇大哭，把脑袋躲在她的怀里，好象要钻进她的胸口中去似的，那个妇女用胳膊搂着他，安慰着他，尽管她自己的脸色也吓得发青。她一度用自己的胳膊尽可能地掩护着他，仿佛她以为自己的胳膊能够抵御子弹不伤他的身体似的。接着直升飞机在他们中间投了一颗二十公斤的炸弹，引起可怕的爆炸，救生艇四分五裂，成为碎片。接着出现一个很精采的镜头一个孩子的胳膊举了起来越举越高越举越高一直到了天空中一定有架机头装着摄影机的直升飞机跟着他的胳膊，在党员座中间发出了很多的掌声但是在无产座部分有个妇女突然吵了起来大声说他们不应该在孩子们面前放映这部电影他们在孩子们面前放映这部电影是不对的最后警察把她赶了出去我想她不致于会遇到什么不愉快的结果无产者说些什么没有人会放在心上典型的无产者反应他们决不会——

　　温斯顿停下了笔，一半是因为他感到手指痉挛。他也不知道是什么东西使他一泻千里地写出这些胡说八道的话来。

　　但奇怪的事情是，他在写的时候，有一种完全不同的记忆在他的思想中明确起来，使他觉得自已有能力把它写下来。他现在认识到，这是因为有另一件事情才使他突然决定今天要回家开始写日记。

　　如果说，这样一件模模糊糊的事也可以说是发生的话，这件事今天早上发生在部里。

　　快到十一点的时候，在温斯顿工作的纪录司，他们把椅子从小办公室拖出来，放在大厅的中央，放在大电幕的前面，准备举行两分钟仇恨。温斯顿刚刚在中间一排的一张椅子上坐下来，有两个他只认识脸孔、却从来没有讲过话的人意外地走了进来。其中有一个是他常常在走廊中遇到的一个姑娘。

　　他不道她的名字，但是他知道她在小说司工作。由于他有时看到她双手沾油，拿着扳钳，她大概是做机械工的，拾掇那些小说写作机器。她是个年约二十七岁、表情大胆的姑娘，浓浓的黑发，长满雀斑的脸，动作迅速敏捷，象个运动员。她的工作服的腰上重重地围了一条猩红色的狭缎带，这是青年反性同盟的标志，围的不松不紧，正好露出她的腰部的苗条。温斯顿头一眼看到她就不喜欢她。他知道为什么原因。这是因为她竭力在自己身上带着一种曲棍球场、冷水浴、集体远足、总的来说是思想纯洁的味道。几乎所有的女人他都不喜欢，特别是年轻漂亮的。总是女人，尤其是年轻的女人，是党的最盲目的拥护者，生吞活剥口号的人，义务的密探，非正统思想的检查员。但是这个女人使他感到比别的更加危险。有一次他们在走廊里遇到时，她很快地斜视了他一眼，似乎看透了他的心，刹那间他充满了黑色的恐惧。他甚至想到这样的念头：她可能是思想警察的特务。不错，这是很不可能的。但是只要她在近处，他仍有一种特别的不安之感。这种感觉中掺杂着敌意．也掺杂着恐惧。

　　另外一个人是个叫奥勃良的男人，他是核心党员，担任的职务很重要，高高在上，因此温斯顿对他职务的性质只有一种很模糊的概念。椅子周围的人一看到核心党员的黑色工作服走近时，都不由得肃静下来。奥勃良是个体格魁梧的人，脖子短粗，有着一张粗犷残忍、兴高采烈的脸。尽管他的外表令人望而生畏，他的态度却有一定迷人之处。他有一个小动作奇怪地使人感到可亲，那就是端正一下鼻梁上的眼镜；也很难说清楚，这奇怪地使人感到很文明。如果有人仍旧有那样想法的话，这个姿态可能使人想到一个十八世纪的绅士端出鼻烟匣来待客。温斯顿大概在十多年来看到过奥勃良十多次。他感到对他特别有兴趣，这并不完全是因为他对奥勃良彬彬有礼的态度和拳击师的体格的截然对比感到有兴趣。

　　更多的是因为他心中暗自认为——也许甚至还不是认为，而仅仅是希望——奥勃良的政治信仰不完全是正统的。他脸上的某种表情使人无法抗拒地得出这一结论。而且，表现在他脸上的，甚至不是不正统，而干脆就是智慧。不过无论如何，他的外表使人感到，如果你能躲过电幕而单独与他在一起的话，他是个可以谈谈的人。温斯顿从来没有做过哪怕是最轻微的努力来证实这种猜想；说真的，根本没有这样做的可能。现在，奥勃良瞥了一眼手表，看到已经快到十一点了，显然决定留在纪录司，等两分钟仇恨结束。他在温斯顿那一排坐了下来，相隔两把椅子。中间坐的是一个淡茶色头发的小女人，她在温斯顿隔壁的小办公室工作。那个黑头发的姑娘坐在他们背后一排。

　　接着，屋子那头的大电幕上突然发出了一阵难听的摩擦声，仿佛是台大机器没有油了一样。这种噪声使你牙关咬紧、毛发直竖。仇恨开始了。

　　象平常一样，屏幕上闪现了人民公敌爱麦虞埃尔果尔德施坦因的脸。观众中间到处响起了嘘声。那个淡茶色头发的小女人发出了混杂着恐惧和厌恶的叫声。果尔德施坦因是个叛徒、变节分子，他一度(那是很久以前了，到底多久，没有人记得清楚)是党的领导人物之一，几乎与老大哥本人平起平坐，后来从事反革命活动，被判死刑，却神秘地逃走了，不知下落。两分钟仇恨节目每天不同，但无不以果尔德施坦因为其重要人物。他是头号叛徒，最早污损党的纯洁性的人。后来的一切反党罪行、一切叛国行为、破坏颠覆、异端邪说、离经叛道都是直接起源于他的教唆。反正不知在什么地方，他还活着，策划着阴谋诡计；也许是在海外某个地方，得到外国后台老板的庇护；也许甚至在大洋国国内某个隐蔽的地方藏匿着——有时就有这样的谣传。

　　温斯顿眼睛的隔膜一阵抽搐。他看到果尔德施坦因的脸时不由得感到说不出的滋味，各种感情都有，使他感到痛苦。

　　这是一张瘦削的犹太人的脸，一头蓬松的白发，小小的一撮山羊胡须——一张聪明人的脸庞，但是有些天生的可鄙，长长的尖尖的鼻子有一种衰老性的痴呆，鼻尖上架着一副眼镜。这张脸象一头绵羊的脸，它的声音也有一种绵羊的味道。

　　果尔德施坦因在对党进行他一贯的恶毒攻击，这种攻击夸张其事，不讲道理，即使一个儿童也能一眼看穿，但是听起来却有似乎有些道理，使你觉得要提高警惕，别人要是没有你那么清醒的头脑，可能上当受骗。他在谩骂老大哥，攻击党的专政，要求立即同欧亚国媾和，主张言论自由、新闻自由、集会自由、思想自由，歇斯底里地叫嚷说革命被出卖了——

　　所有这一切的话都是用大字眼飞快地说的，可以说是对党的演说家一贯讲话作风的一种模仿，甚至还有一些新话的词汇；说真的，比任何党员在实际生活中一般使用的新话词汇还要多。在他说话的当儿，唯恐有人会对果尔德施坦因的花言巧语所涉及的现实有所怀疑，电幕上他的脑袋后面有无穷无尽的欧亚国军队列队经过——一队又一队的结实的士兵蜂拥而过电幕的表面，他们的亚细亚式的脸上没有表情，跟上来的是完全一样的一队士兵。这些士兵们的军靴有节奏的踩踏声衬托着果尔德施坦因的嘶叫声。

　　仇恨刚进行了三十秒钟，屋子里一半的人中就爆发出控制不住的愤怒的叫喊。电幕上扬扬自得的羊脸，羊脸后面欧亚国可怕的威力，这一切都使人无法忍受；此外，就凭果尔德施坦因的脸，或者哪怕只想到他这个人，就自动的产生恐惧和愤怒。不论同欧亚国相比或东亚国相比，他更经常的是仇恨的对象，因为大洋国如果同这两国中的一国打仗，同另外一国一般总是保持和平的。但是奇怪的是，虽然人人仇恨和蔑视果尔德施坦因，虽然每天，甚至一天有上千次，他的理论在讲台上、电幕上、报纸上、书本上遭到驳斥、抨击、嘲笑，让大家都看到这些理论是多么可怜的胡说八道，尽管这样，他的影响似乎从来没有减弱过。总是有傻瓜上当受骗。思想警察没有一天不揭露出有间谍和破坏分子奉他的指示进行活动。他成了一支庞大的隐蔽的军队的司令，这是一帮阴谋家组成的地下活动网，一心要推翻国家政权。它的名字据说叫兄弟团，谣传还有一本可怕的书，集异端邪说之大成，到处秘密散发，作者就是果尔德施坦因。这本书没有书名。大家提到它时只说那本书。不过这种事情都是从谣传中听到的。任何一个普通党员，只要办得到，都是尽量不提兄弟团或那本书(thebook)的。

　　仇恨到了第二分钟达到了狂热的程度。大家都跳了起来，大声高喊，要想压倒电幕上传出来的令人难以忍受的羊叫一般的声音。那个淡茶色头发的小女人脸孔通红，嘴巴一张一闭，好象离了水的鱼一样。甚至奥勃良的粗犷的脸也涨红了。他直挺挺地坐在椅上，宽阔的胸膛胀了起来，不断地战栗着，好象受到电流的袭击。温斯顿背后的黑头发姑娘开始大叫“猪猡！猪猡！猪猡！”她突然拣起一本厚厚的新话词典向电幕扔去。它击中了果尔德施坦因的鼻子，又弹了开去，他说话的声音仍旧不为所动地继续着。温斯顿的头脑曾经有过片刻的清醒，他发现自已也同大家一起在喊叫，用鞋后跟使劲地踢着椅子腿。两分钟仇恨所以可怕，不是你必须参加表演，而是要避不参加是不可能的。不出三十秒钟，一切矜持都没有必要了。一种夹杂着恐惧和报复情绪的快意，一种要杀人、虐待、用大铁锤痛打别人脸孔的欲望，似乎象一股电流一般穿过了这一群人，甚至使你违反本意地变成一个恶声叫喊的疯子。然而，你所感到的那种狂热情绪是一种抽象的、无目的的感情，好象喷灯的火焰一般，可以从一个对象转到另一个对象。因此，有一阵子，温斯顿的仇恨并不是针对果尔德施坦因的，而是反过来转向了老大哥、党、思想警察；在这样的时候，他打从心跟里同情电幕上那个孤独的、受到嘲弄的异端分子，谎话世界中真理和理智的唯一卫护者。可是一会儿他又同周围的人站在一起，觉得攻击果尔德施坦因的一切的话都是正确的。在这样的时刻，他心中对老大哥的憎恨变成了崇拜，老大哥的形象越来越高大，似乎是一个所向无故、毫无畏惧的保护者，象块巨石一般耸立于从亚洲蜂拥而来的乌合之众之前，而果尔德施坦因尽管孤立无援，尽管对于是否有他这个人的存在也有怀疑，却似乎是一个阴险狡诈的妖物，光凭他的谈话声音也能够把文明的结构破坏无遗。

　　有时候，你甚至可以自觉转变自己仇恨的对象。温斯顿突然把仇恨从电幕上的脸孔转到了坐在他背后那个黑发女郎的身上，其变化之迅速就象做恶梦醒来时猛的坐起来一样。一些栩栩如生的、美丽动人的幻觉在他的心中闪过。他想象自己用橡皮棍把她揍死，又把她赤身裸体地绑在一根木桩上，象圣塞巴斯蒂安一样乱箭丧身。在最后高潮中，他污辱了她，割断了她的喉管。而且，他比以前更加明白他为什么恨她。

　　他恨她是因为她年青漂亮，却没有性感，是因为他要同她睡觉但永远不会达到目的，是因为她窈窕的纤腰似乎在招引你伸出胳膊去搂住她，但是却围着那条令人厌恶的猩红色绸带，那是咄咄逼人的贞节的象征。

　　仇恨达到了最高潮。果尔德施坦因的声音真的变成了羊叫，而且有一度他的脸也变成了羊脸。接着那头羊脸又化为一个欧亚国的军人，高大吓人，似乎在大踏步前进，他的轻机枪轰鸣，似乎有夺幕而出之势，吓得第一排上真的有些人从坐着的椅子中来不及站起来。但是就在这一刹那间，电幕上这个敌人已化为老大哥的脸，黑头发，黑胡子，充满力量，镇定沉着，脸庞这么大，几乎占满了整个电幕，他的出现使大家放心地深深松了一口气。没有人听见老大哥在说什么。他说的只是几句鼓励的话，那种话一般都是在战斗的喧闹声中说的，无法逐宇逐句听清楚，但是说了却能恢复信心。接着老大的脸又隐去了，电幕上出现了用黑体大写字母写的党的三句口号：

　　战争即和平自由即奴役无知即力量。

　　但是老大哥的脸似乎还留在电幕上有好几秒钟，好象它在大家的视网膜上留下的印象太深了，不能马上消失似的。那个淡茶色头发的小女人扑在她前面一排的椅子背上。她哆哆嗦嗦地轻轻喊一声好象“我的救星！”那样的话，向电幕伸出双臂。接着又双手捧面。很明显，她是在做祷告。

　　这时，全部在场的人缓慢地、有节奏地、深沉地再三高叫“B－B！……B-B！……B-B！”*他们叫得很慢，在第一个B和第二个B之间停顿很久。这种深沉的声音令人奇怪地有一种野蛮的味道，你仿佛听到了赤脚的踩踏和铜鼓的敲打。他们这样大约喊了三十秒钟。这种有节奏的叫喊在感情冲动压倒一切的时候是常常会听到的。这一部分是对老大哥的英明伟大的赞美，但更多的是一种自我催眠，有意识地用有节奏的闹声来麻痹自已的意识。温斯顿心里感到一阵凉。在两分钟的仇恨中，他无法不同大家一起梦呓乱语，但是这种野兽般的“B-B！……B-B！”的叫喊总使他充满了恐惧。当然，他也和大家一起高喊：不那么做是办不到的。掩饰你真实的感情，控制你脸部的表情，大家做什么你就做什么，这是一种本能的反应。但是有那么一两秒钟的时间里，他的眼睛里的神色很可能暴露了他自己。正好是在这一刹那，那件有意义的事情发生了——如果说那件事情真的发生了的话。

　　(*英语“老大哥”的第一个字母——译注)

　　原来在瞬息间他同奥勃良忽然眼光相遇。奥勃良这时已经站了起来。他摘下了眼镜，正要用他一贯的姿态把眼镜放到鼻梁上去。就在这一刹那之间，他们两人的眼光相遇了，在这相遇财刻，温斯顿知道——是啊，他知道(knew)！——奥勃良心里想的同他自己一样。他们两人之间交换了一个无可置疑的信息。好象他们两人的心打了开来，各人的思想通过眼光而流到了对方的心里。“我同你一致，”奥勃良似乎这样对他说。“我完全知道你的想法．你的蔑视、仇恨、厌恶，我全都知道。不过别害怕，我站在你的一边！”但是领悟的神情一闪即逝，奥勃良的肠又象别人的脸一样令人莫测高深了。

　　情况就是这样，他已经在开始怀疑，是不是真的发生过这样的情况，这辞事情是从来不会有后继的，唯一结果不过是在他的心中保持这样的信念，或者说希望：除了他自己以外也有别人是党的敌人。也许，说什么普遍存在着地下阴谋的谣言是确实的也说不定，也许真的有兄弟团的存在！尽管有不断的逮捕、招供和处决，仍不可能有把握地说，兄弟团不只是个谣言面已。他有时相信，有时不相信。没有任何证据，只是一些过眼即逝的现象，可能有意义也可能没有意义：一鳞半爪偶然听来的谈话，厕所墙上的隐隐约约的涂抹——甚至有一次两个素不相识的人相遇时手中一个小动作使人觉得好象他们是在打暗号。这都是瞎猜：很可能这一切都是他瞎想出来的。他对奥勃良不再看一眼就回到他的小办公室去了。他一点也没有想到要追踪他们刚才这短暂的接触。

　　即使他知道应该怎么办，这样做的危险也是无法想象的。他们不过是在一秒钟、两秒钟里交换了明白的眼光，事情就到此为止了。但是即使这样，在这样自我隔绝的孤独的生活环境中，这也是一件意义重大的事。

　　温斯顿挺直腰板，坐了起来。他打了一个嗝。杜松子酒的劲头从他肚子里升了起来。

　　他的眼光又回到本子上。他发现他在无可奈何地坐着胡思乱想的时候，他也一直在写东西，好象是自发的动作一样。而且笔迹也不是原来的那样歪歪斜斜的笨拙笔迹了。他的笔在光滑的纸面上龙飞凤舞，用整齐的大写字母写着——

　　打倒老大哥打倒老大哥打倒老大哥打倒老大哥打倒老大哥

　　一遍又一遍地写满了半页纸。

　　他禁不住感到一阵恐谎。其实并无必要，因为写这些具体的字并不比开始写日记这一行为更加危险；但是有一阵子他真想把这些涂抹了的纸页撕了下来，就此作罢。

　　但是他没有这样做，因为他知道这没有用。不论他是写打倒老大哥，还是他没有写，并没有什么不同。不论他是继续写日记，还是他没有继续写，也没有什么不同。思想警察还是会逮到他的。他已经犯了——即使他没有用笔写在纸上，也还是犯了的——包含一切其他罪行的根本大罪。这明做思想罪。思想罪可不是能长期隐匿的。你可能暂时能躲避一阵，甚至躲避几年，但他们迟早一定会逮到你。

　　总是在夜里——逮捕总是在夜里进行的。突然在睡梦中惊醒，一只粗手捏着你的肩膀，灯光直射你的眼睛，床边围着一圈凶狠的脸孔。在绝大多数情况下不举行审讯，不报道逮捕消息，人就是这么销声匿迹了，而且总是在夜里。你的名字从登记册上除掉了，你做过的一切事情的记录都除掉了，你的一度存在也给否定了，接着被遗忘了。你被取消，消灭了：通常用的字眼是化为乌有(vaporized)。

　　他忽然象神经病发作一样，开始匆忙地乱涂乱划起来：

　　他们会枪毙我我不在乎他们会在我后脑勺打一枪我不在乎打倒老大哥他们总是在后脑勺给你一枪我不在乎打倒老大哥——

　　他在椅子上往后一靠，有点为自已感到难为情，放下了笔。接着他又胡乱地写起来。这时外面传来一下敲门声。

　　已经来了！他象只耗子似的坐着不动，满心希望不论是谁敲门，敲了一下就会走开。但是没有，门又敲了一下。迟迟不去开门是最糟糕的事情。他的心怦怦的几乎要跳出来，但是他的脸大概是出于长期的习惯却毫无表情。他站了起来，脚步沉重地向门走去。

**Reply by hktxt** [reply]
要不，咱在这里连载？

**Reply by itsoursduty** [reply]
有啥用... 懂得人不需要看 不懂的人不想看

**Reply by zhhrz** [reply]
@hktxt 连载吧，还是有人想看的
